data/human/human missions.txt

```diff
@@ -6849,6 +6849,133 @@ mission "FW Refugees to Humanika"
 		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`
 
+
+mission "Derelict Informer"
+	minor
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "uninhabited"
+	to offer
+		random < 4
+		"days since start" > 30
```

Comment:
Actually Intro 0 doesn't seem to be priority like the rest? Which is the one that would affect this, that seems like a separate issue that needs to be fixed though.

Reply:
Ah, yeah, because the first intro mission is actually a landing mission (which can't have priority). This mission offers in the spaceport, which means the intro mission being a landing mission won't block it.